action.yml

```diff
@@ -8,7 +8,12 @@ inputs:
 runs:
   using: composite
   steps:
-    - run: |-
-        bash <(curl -s https://raw.githubusercontent.com/ory/ci/master/src/scripts/install/prettier.sh)
-        npm run $INPUT_NPMRUNARGS
+    - name: Install Prettier if it doesn't exist
+      run: |-
+        if [ ! -f node_modules/.bin/prettier ]; then
+          bash <(curl -s https://raw.githubusercontent.com/ory/ci/master/src/scripts/install/prettier.sh)
```

Comment:
If we use npx it will either install it or use the local version :)

Reply:
That's a very good point! This makes me wonder how this GitHub Action is actually used and what that means for its implementation. I will close this PR, will gather data points while I set up format checks in a few repos, and will be back with a better update for this GitHub Action.